[Alerts][Docs] Added API documentation for alerts plugin (#91067)
elastic/kibana: docs/api/alerts/unmute_all.asciidoc

[[alerts-api-unmute-all]]
=== Unmute all alert instances API
++++
<titleabbrev>Unmute all alert instances</titleabbrev>
++++

Unmute all {kib} alert instances.

[[alerts-api-unmute-all-request]]
==== Request

`POST <kibana host>:<port>/api/alerts/alert/<id>/_unmute_all`

[[alerts-api-unmute-all-path-params]]
==== Path parameters

`id`::

>   (Required, string) The alert ID that you want to unmute all instances.

Reviewer: ```suggestion
  (Required, string) The Alert ID of the Alert whose instances we wish to unmute.
```
